src/app/debug/table/table-settings-modal/table-settings-modal.component.html

```diff
@@ -14,7 +14,7 @@ <h5 class="modal-title" id="settingsModal">Options</h5>
         <h5>Debug tree</h5>
         <div class="checkbox " title="When checked, file tree shows multiple files">
           <label for="show-multiple">Show multiple files in debug file-tree</label>
-          <input class="btn btn-info my-2 p-1 w-fit" data-cy-toggle-show-amount id="show-multiple"
+          <input class="btn btn-info my-2 p-1 w-fit" data-cy-settings-modal="showAmount" id="show-multiple"
```

Comment:
Modal mag je hier weglaten
```suggestion
          <input class="btn btn-info my-2 p-1 w-fit" data-cy-settings="showAmount" id="show-multiple"
```